Cargo.toml

```diff
@@ -13,22 +13,34 @@ rayon = ["blake3/rayon"]
 
 [dependencies]
 anyhow = "1.0.69"
-axum = "0.6.9"
+#axum = "0.6.9"
+axum = { version = "0.6.9", features = ["headers"] }
+
+
 axum-macros = "0.3.4"
 bao = "0.12.1"
 blake3 = "1.3.3"
 bytes = "1.4.0"
-carbonado = "0.3.0-rc.8"
+
+#carbonado = "0.3.0-rc.8"
+
+carbonado = { path = "/Users/wesleyparr/Projects/carbonado/carbonado" }
+
 chrono = "0.4.23"
 clap = { version = "4.1.8", features = ["derive"] }
 directories = "5.0.1"
 flexi_logger = { version = "0.25.2", features = ["colors"] }
 flexi_syslog = "0.5.2"
+futures = "0.3.28"
```

Comment:
We shouldn't need this except for very specific cases, try to find a way to work around it